assets/scss/components/_button.scss

```diff
@@ -67,9 +67,9 @@ Applies button color-related styles using a mixin.
 ```
 */
 @mixin buttonColor(
-	$bgColor: $C_buttonBG,
-	$hoverColor: opacify($bgColor, .05),
-	$activeColor: opacify($hoverColor, .1),
+	$bgColor: $C_blue,
+	$hoverColor: darken($bgColor, 5%),
+	$activeColor: darken($bgColor, 12%),
```

Comment:
pref of percent over decimal?

Reply:
the `darken` function only takes a percentage